app/assets/javascripts/controllers/home/home.coffee

```diff
@@ -1 +1,8 @@
 @cooking.controller 'homeCtrl', ($scope) ->
+  $scope.recipes = [
+    {
+      id:1,
+      title:"recipe 1",
+      description:"SmartOS images come with pkgsrc pre-installed. On other illumos distributions, first install pkgsrc, then you may install the binary package as normal: SmartOS images come with pkgsrc pre-installed. On other illumos distributions, first install pkgsrc, then you may install the binary package as normal:",
+      username:"Mohamed Belaid"
+      }]
```

Comment:
create a service/factory that returns this object.
And better, put this in a json file then do a http call (to this file to get data) to simulate the call to the api. 
In the future we juste replace the file url with the api url.